src/update-readme.ts

```diff
@@ -49,7 +54,9 @@ export function generateReadme(
     content =
       content.slice(0, start) +
       `${startString}\n` +
-      '<h4><a href="https://github.com/my-badges/my-badges">My Badges</a></h4>\n\n' +
+      (!omitheader
```

Comment:
Let's use var here:

```js 
const headerHtml = omitheader ? '' : '<...>'
```

Reply:
`pick / omit` and `omitheader`. Maybe just `noheader`?

CC @antonmedv

Reply:
noheader better